### PR TITLE
Add MATLAB docs

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

---

**Related issue :**

### CODE REVIEW

File: content/en/users/notebooks/kernels/_index.md [hunk 1]
```diff
@@ -1,50 +1,219 @@
...
 description: >
-  Available Kernels in EGI Notebooks
+  Available Environments in EGI Notebooks
 weight: 5
 ---
 
 Jupyter is an extensible environment that supports different programming
-languages. Each language is available as a different
-[kernel](https://jupyter.readthedocs.io/en/latest/projects/kernels.html).
+languages. For the EGI service we have enabled two differnet environments:
```
Comment (giannidallatorre) [inline content/en/users/notebooks/kernels/_index.md:10]: "differnet" Typo

File: content/en/users/notebooks/kernels/_index.md [hunk 2]
```diff
@@ -1,50 +1,219 @@
...
+
+After logging into the service, you will be shown a form for selecting the
+environment, pick the desired one and click start
+
+![environmet selection](notebooks-environments.png)
+
+## Default environment
+
+The default environment includes a set of
+[kernels](https://jupyter.readthedocs.io/en/latest/projects/kernels.html). that
```
Comment (giannidallatorre) [inline content/en/users/notebooks/kernels/_index.md:24]: "that are automatically built"

File: content/en/users/notebooks/kernels/_index.md [hunk 3]
```diff
@@ -1,50 +1,219 @@
...
-## R
+## MATLAB environment
 
-The R programming language with several packages from the R ecosystem as
-provided by
-[jupyter/r-notebook](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#jupyter-r-notebook)
-and some extra libraries.
+EGI offers MATLAB on the cloud using the
+[MATLAB Integration for Jupyter](https://www.mathworks.com/products/reference-architectures/jupyter.html)
+and share data and code with other EGI users. MATLAB is also available directly
```
Comment (giannidallatorre) [inline content/en/users/notebooks/kernels/_index.md:55]: shares

File: content/en/users/notebooks/kernels/_index.md [hunk 4]
```diff
@@ -1,50 +1,219 @@
...
+   ![matlab-ide](matlab-ide.png)
+
+1. To facilitate sharing your research output, you can use
+   [MATLAB Live Scripts](https://www.mathworks.com/help/matlab/matlab_prog/create-live-scripts.html)
+   combining rich text, equations, images, code and inline output all in one
+   document. Live Scripts can be accessed from the
+   [Live Editor](https://www.mathworks.com/products/matlab/live-editor.html)
+   tab. Here is an example of a tutorial Live Script from the EGI webinar
+   showing steps required to analyze some public COVID-19 data with MATLAB [add
+   link to Live Script here]. Some other examples of Live Scripts can be found
```
Comment (giannidallatorre) [inline content/en/users/notebooks/kernels/_index.md:130]: Missing link

**Review by giannidallatorre — APPROVED** [review]

File: content/en/users/notebooks/kernels/_index.md (hunk 1 above)
Comment (enolfc) [reply content/en/users/notebooks/kernels/_index.md:10]:
```suggestion
languages. For the EGI service we have enabled two different environments:
```

File: content/en/users/notebooks/kernels/_index.md (hunk 2 above)
Comment (enolfc) [reply content/en/users/notebooks/kernels/_index.md:24]:
```suggestion
[kernels](https://jupyter.readthedocs.io/en/latest/projects/kernels.html) that
```

File: content/en/users/notebooks/kernels/_index.md [hunk 5]
```diff
@@ -1,50 +1,219 @@
...
+After logging into the service, you will be shown a form for selecting the
+environment, pick the desired one and click start
+
+![environmet selection](notebooks-environments.png)
+
+## Default environment
+
+The default environment includes a set of
+[kernels](https://jupyter.readthedocs.io/en/latest/projects/kernels.html). that
+are automaticall build from the
```
Comment (enolfc) [inline content/en/users/notebooks/kernels/_index.md:25]:
```suggestion
are automatically built from the
```

File: content/en/users/notebooks/kernels/_index.md (hunk 3 above)
Comment (enolfc) [reply content/en/users/notebooks/kernels/_index.md:55]:
```suggestion
and shares data and code with other EGI users. MATLAB is also available directly
```

File: content/en/users/notebooks/kernels/_index.md (hunk 4 above)
Comment (enolfc) [reply content/en/users/notebooks/kernels/_index.md]: Fixed

**Review by andrea-manzi — APPROVED** [review]
LGTM!

**Pull request merged**